pkg/cmd/pr/shared/survey.go

```diff
@@ -110,10 +110,19 @@ func BodySurvey(p Prompt, state *IssueMetadataState, templateContent string) err
 	return nil
 }
 
-func TitleSurvey(p Prompt, state *IssueMetadataState) error {
-	result, err := p.Input("Title", state.Title)
-	if err != nil {
-		return err
+func TitleSurvey(p Prompt, io *iostreams.IOStreams, state *IssueMetadataState) error {
+	var err error
+	result := ""
+	for result == "" {
+		result, err = p.Input("Title (required)", state.Title)
+		if err != nil {
+			return err
+		}
+		if result == "" {
+			colorizeRed := io.ColorScheme().ColorFromString("red")
+			msg := fmt.Sprintf("%s Title cannot be blank.", colorizeRed("X"))
+			fmt.Fprintln(io.ErrOut, msg)
+		}
```

Comment:
I think you mentioned this during discussion, but would love to see if we can get the following working being more `gh` idiomatic:

```suggestion
		if result == "" {
			fmt.Fprintf(io.ErrOut, "%s Title cannot be blank\n", cs.FailureIcon())
		}
```

Reply:
Seems to work 🤔 

```shell
andyfeller@Andys-MBP:cli/cli ‹jtmcg/9698*›$ git diff | cat
diff --git a/pkg/cmd/pr/shared/survey.go b/pkg/cmd/pr/shared/survey.go
index 3d53be4c..ce38535d 100644
--- a/pkg/cmd/pr/shared/survey.go
+++ b/pkg/cmd/pr/shared/survey.go
@@ -119,9 +119,7 @@ func TitleSurvey(p Prompt, io *iostreams.IOStreams, state *IssueMetadataState) e
 			return err
 		}
 		if result == "" {
-			colorizeRed := io.ColorScheme().ColorFromString("red")
-			msg := fmt.Sprintf("%s Title cannot be blank.", colorizeRed("X"))
-			fmt.Fprintln(io.ErrOut, msg)
+			fmt.Fprintf(io.ErrOut, "%s Title cannot be blank\n", io.ColorScheme().FailureIcon())
 		}
 	}
 
andyfeller@Andys-MBP:cli/cli ‹jtmcg/9698*›$ make
go build -trimpath -ldflags "-X github.com/cli/cli/v2/internal/build.Date=2024-10-17 -X github.com/cli/cli/v2/internal/build.Version=v2.58.0-105-gc1897f31 " -o bin/gh ./cmd/gh
andyfeller@Andys-MBP:cli/cli ‹jtmcg/9698*›$ ./bin/gh issue create              

Creating issue in cli/cli

? Title (required) 
X Title cannot be blank
? Title (required) 
X Title cannot be blank
? Title (required) 
X Title cannot be blank
? Title (required) 
X Title cannot be blank
? Title (required) 
```

Reply:
Iiiiiiinteresting.... So apparently using the `io.ColorScheme().FailureIcon()` as the argument for the formatted string is what's making the difference, here.

I also tried 

```golang
    colorizeRed := io.ColorScheme().ColorFromString("red")
    fmt.Fprintln(io.ErrOut, "%s Title cannot be blank.", colorizeRed("X"))
```

And that was not rendering the `X Title cannot be blank` bit during the loop. However, your code does fix it 👍 I definitely prefer what you've done, here. Thanks for the assist 🙌